build/mac/cross-compile/README.md

```diff
@@ -0,0 +1,113 @@
+# Cross-compiling Brave
+
+The code in this folder makes it possible to compile Brave for macOS in Linux.
+
+## Initialization
+
+Use the following commands to prepare the build:
+
+```
+npm install
+npm run init -- --target_os=mac --target_arch=x64
+```
+
+## Compiling Brave
+
+After the above, you can compile Brave for macOS with the following command:
+
+```
+npm run build -- --target_os=mac --target_arch=x64
+```
+
+`Static` builds also work:
+
+```
+npm run build -- Static --target_os=mac --target_arch=x64
+```
+
+For `Release`, you need to disable dSYM generation and symbol stripping:
```

Comment:
> you need to disable dSYM generation and symbol stripping

why?

Reply:
It doesn't "just work" and I haven't yet had time to implement this. It can be done in a follow-up PR. I added a clarifying statement about this in c825f210e9a7b930b9969004217f2064ff6e1676.